src/solvers/radau5.py

```diff
@@ -117,14 +121,106 @@ def __init__(self, problem):
         self._type = '(explicit)'
         self._event_info = None
         self._werr = N.zeros(self._leny)
+
+    def _get_linear_solver(self):
+        return self.options["linear_solver"]
+
+    def _set_linear_solver(self, linear_solver):
+        """
+        Which type of linear solver to use, "DENSE" or "SPARSE"
+        
+            Parameters::
+            
+                linear_solver
+                                - Default "DENSE"
+                            
+                                - needs to be either "DENSE" or "SPARSE"
+        """
+        try:
+            linear_solver.upper()
```

Comment:
since `linear_solver.upper()` is called multiple times here, perhaps we should set a variable to it, i.e. `linear_solver_upper = linear_solver.upper()`

Reply:
Fixed, also for 'implementation'